### PR TITLE
Add option to specify host address

### PR DESCRIPTION
**Pull request by GlancingMind**
'--host'-cmdline-option is added to the webui command.

Previously, the WebUI couldn't be hosted inside of a container. As the
WebUI-server only listend per default to localhost and there was no
option to change the address, the server should listend to. This means,
that the WebUI was only reachable from localhost. So only from inside of
the container but never from outside.
The '--host'-option allows to set the IP address or a hostname
which the WebUI-server should listen to. E.g. by setting 0.0.0.0 or ::
as address.

Update documentation for new option.

Update shell completion for new option.

**Somehow compilation adds another go-gitlab version.**
Might be because of https://github.com/MichaelMure/git-bug/pull/570#issue-573242380

### CODE REVIEW

File: commands/webui.go [hunk 1]
```diff
@@ -54,6 +57,7 @@ Available git config:
 	flags := cmd.Flags()
 	flags.SortFlags = false
 
+	flags.StringVar(&options.host, "host", "127.0.0.1", "Network address or hostname to listen to (default 127.0.0.1)")
```
Comment (MichaelMure) [inline commands/webui.go:60]:
```suggestion
	flags.StringVar(&options.host, "host", "127.0.0.1", "Network address or hostname to listen to (default to 127.0.0.1)")
```

**Comment by MichaelMure** [comment]
Closing in favor of https://github.com/MichaelMure/git-bug/pull/584

I wanted to tweak the help messages but I couldn't push on your fork to just add to your PR, so I made a new (with you still as the author of the commit).

**Comment by MichaelMure** [comment]
> **Somehow compilation adds another go-gitlab version.**
> Might be because of [#570 (comment)](https://github.com/MichaelMure/git-bug/pull/570#issue-573242380)

That's correct. Annoyingly dependabot doesn't fully update the `go.sum` file so the next to compile get to commit those changes...

**Comment by GlancingMind** [comment]
> Closing in favor of #584
> 
> I wanted to tweak the help messages but I couldn't push on your fork to just add to your PR, so I made a new (with you still as the author of the commit).

Oh sorry for the inconvenience. I unchecked "Allow edits and access to secrets by maintainers" :grimacing:.

**Comment by MichaelMure** [comment]
No worries it was quick. But yeah this options is super convenient for maintainers.

**Pull request closed without merging**